src/components/v5/shared/SearchSelect/types.ts

```diff
@@ -22,10 +23,10 @@ export interface SearchSelectOptionProps {
 }
 
 export interface SearchSelectOption {
-  label: MessageDescriptor | string;
-  value: string;
+  label?: MessageDescriptor | string | null;
```

Comment:
Does this make any sense to have an option without label?

Reply:
yes you're right but in this hook `useGetVerifiedMembersQuery` label can be string | null | undefined

Reply:
So maybe you should filter options that are not possible to give them value? You can use `reduce` for that